pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -345,6 +347,15 @@ type CephxConfig struct {
 	// +kubebuilder:validation:Maximum=4294967295
 	// +kubebuilder:validation:XValidation:message="keyGeneration cannot be decreased",rule="self >= oldSelf"
 	KeyGeneration uint32 `json:"keyGeneration,omitempty"`
+
+	// PriorKeyCount available for components that use overlapping key rotation or
+	// for non-local Ceph daemon eg. CSI, RBD/CephFS mirror.
+	// This tells Rook how many prior keys to keep active.
+	// Generally, this would be set to 1 to allow for a migration period for applications.
+	// If desired, set this to 0 to delete prior keys after migration
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Enum=0;1
```

Comment:
We don't want to limit this to just 0 and 1. Let's use the same min/max as `KeyGeneration`. Or possibly I think we could limit this to something like 5 or 10. But 1 seems like it could be too few.